schema/elasticsearch/v7/visibility/versioned/v0/index_template.json

```diff
@@ -0,0 +1,63 @@
+{
```

Comment:
I decided to keep history of index_templates in `versioned` directory. Mostly because I added reindex script and it is not clear what it reindex to what w/o having both old and new index templates handy.

I plan to use major version for every schema change which requires reindex (unlikely to happen again) and minor for changes that can be done w/o it (adding new indexed field, for instance).

Reply:
The current structure require to copy all old versions once a new version is created? For example, once you have a v8, you would need to have v8/visibility/versioned/[v0, v1, v2, ...].
Can we use the same structure as in db schema? so the structure looks like:
schema/elasticsearch/visibility/index_template.json <-- current latest version
schema/elasticsearch/visibility/versioned/v0/index_template.json <-- v0 version
schema/elasticsearch/visibility/versioned/v1/index_template.json <-- v1 version.

Reply:
The key difference here is that ES v6 schema template is different from ES v7 one and they can't be combined. Hopefully v8 won't have breaking changes for the schema template and will continue to use v7 one.